csv2sql.drush.inc

```diff
@@ -45,46 +46,53 @@ function drush_csv2sql($csv_path) {
     return;
   }
 
-  if (!file_exists($csv_path)) {
+  // Check if the command has a file direct path.
+  $files = $csv_path ? array($csv_path) : array();
+  // Check if the command has a directory path to scan.
+  if ($dir = drush_get_option('dir')) {
+    $files = csv2sql_scan_for_csv_files($dir);
+  }
+
+  if (!file_exists($csv_path) && empty($files)) {
```

Comment:
seems wrong. If you have some path-to-file, but file doesn't exist - you will have:
``!file_exists($csv_path)`` - TRUE
``empty($files)`` - FALSE (see https://github.com/Gizra/csv2sql/pull/14/files#diff-7cc856ec386c131f823498e60c7b73acR52)

Thus you won't go inside the `if`